src/main/java/sorald/processor/CollectionIsEmptyProcessor.java

```diff
@@ -0,0 +1,34 @@
+package sorald.processor;
+
+import sorald.annotations.ProcessorAnnotation;
+import spoon.reflect.code.*;
+import spoon.reflect.declaration.CtMethod;
+
+@ProcessorAnnotation(
+        key = 1155,
+        description = "Collection.isEmpty() should be used to test for emptiness")
+public class CollectionIsEmptyProcessor extends SoraldAbstractProcessor<CtBinaryOperator<?>> {
+
+    @Override
+    protected void repairInternal(CtBinaryOperator<?> element) {
+        CtExpression<?> methodCallTarget =
+                ((CtInvocation) element.getLeftHandOperand()).getTarget();
```

Comment:
```suggestion
                ((CtInvocation<?>) element.getLeftHandOperand()).getTarget();
```

Reply:
Done!

Reply:
Nice! Just waiting for CI then I'll merge.